testing_and_setup/compass/ocean/cosine_bell/RMSE/default/compute_rmse.py

```diff
@@ -1,76 +1,85 @@
+#!/usr/bin/env python
+
 import numpy as np
 import xarray as xr
 import matplotlib
 matplotlib.use('Agg')
 import matplotlib.pyplot as plt
-import os
 
 dpi = 200
 
+
 def rmse(resTag):
-# resTag is the resolution to compute RMSE
-# sliceTag is the time slice, assumes 1 hour output and same for all cases
-    fid = open('../../../{}/default/init_step/namelist.ocean'.format(resTag),'r')
+    """
+    resTag is the resolution to compute RMSE
+    """
+
+    fid = open('../../../{}/default/init_step/namelist.ocean'.format(resTag),
+               'r')
```

Comment:
Here and many other places, lines should be less than 80 characters if possible and commas need whtespace after them.